yt/fields/xray_emission_fields.py

```diff
@@ -83,7 +83,7 @@ class XrayEmissivityIntegrator:
     def __init__(self, table_type, redshift=0.0, data_dir=None, use_metals=True):
 
         filename = _get_data_file(table_type, data_dir=data_dir)
-        only_on_root(mylog.info, "Loading emissivity data from %s." % filename)
+        only_on_root(mylog.info, f"Loading emissivity data from {filename}.")
```

Comment:
I think this is a mylog statement?

Reply:
yup, but in this case because it's wrapped, the string formatting is evaluated early no matter what, so may as well use a fstring. :)

Reply:
right. I'm just pointing out a mylog instance since the PR specifically says this shouldn't touch mylog calls.

Reply:
OK if we really want to get pedantic, the *correct* fix should be

```suggestion
        only_on_root(mylog.info, "Loading emissivity data from %s", filename)
```

since `only_on_root` has this signature:

```
def only_on_root(func, *args, **kwargs):
```

Reply:
Isn't being pedantic the whole point here ? 😄

Reply:
Maybe we should just delete this line 😈 😈 😈 😈 😈

Reply:
![image](https://user-images.githubusercontent.com/89019/89803444-74877380-daf8-11ea-9d1d-22ac53cd30ac.png)

Reply:
Hi @cphyc we're all here to offer you only the *best* PR commentary. 🍾 🥂

Reply:
AFAIC Matt won by a landslide.